test/specs/scenarios/document.js

```diff
@@ -425,6 +427,24 @@ export const documentScenarios = async (lang) => {
         verificationComplete.verifyUIElements(copy)
       })
 
+      it('should be taken to the cross-device flow for video capture if there is no camera and docVideo variant requested', async () => {
+        driver.get(`${localhostUrl}?language=${lang}&docVideo=true`)
+        driver.executeScript(
+          'window.navigator.mediaDevices.enumerateDevices = () => Promise.resolve([])'
+        )
+        welcome.continueToNextStep()
+        documentSelector.clickOnPassportIcon()
+        crossDeviceIntro.verifyTitle(copy)
+      })
+
+      // @TODO: remove this test when we fully support docVideo variant for both desktop & mobile web
```

Comment:
Are you going to support both mobile and desktop as part of this PR?

Reply:
For now desktop isn't a priority. OTOH webcam on desktop isn't good enough for the video capture of documents.